src/IO.py

```diff
@@ -228,18 +228,18 @@ def get_last(output_string):
     except:
         return "notDigit"
 
-def writeGenomeNewick(pruferSeq, times, populations, name_file):
+def writeGenomeNewick(pruferSeq, times, populations, name_file, file_path):
     children = find_children(pruferSeq, times)
     root = children[-1][0][0]
     root_time = children[-1][0][1]
 
     result = Vertex(root, root_time, children, populations)
 
-    f_nwk = open(name_file + '_tree.nwk', 'w')
+    f_nwk = open(file_path + '/' + name_file + '_tree.nwk', 'w')
     f_nwk.write(result.get_children(root_time))
```

Comment:
Drive-by comment : better use `with open(...) as f_nwk:`

Reply:
Agreed, didn't want to tamper with the initial Pokrovsky code for too much